src/vos/vos_pool_scrub.c

```diff
@@ -379,8 +379,10 @@ sc_verify_recx(struct scrub_ctx *ctx, d_iov_t *data)
 		bool		 match;
 		daos_size_t	 rec_in_chunk;
 
-		if (sc_cont_is_stopping(ctx))
+		if (sc_cont_is_stopping(ctx))	{
+			D_FREE(csum_buf);
 			return 0;
+		}
```

Comment:
Could we `goto done`?

Reply:
Good suggestion. I will update it. Thank you!

Reply:
Fixed. Thank you!